lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -58,6 +58,18 @@
 
 namespace SPIRV {
 
+namespace {
+std::string to_string(uint32_t Version) {
```

Comment:
@svenvh what do you thing, should we fix clang-tidy suggestion here (and in main branch to be aligned)? to rename `to_string` to `toString`

Reply:
Agree to ignore here and whenever it's needed :) Thanks!